### PR TITLE
Improved Content

### PR DESCRIPTION
**Pull request by GovindaSharma**
## This PR is a:

- [ ] New topic
- [x] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will fixed some word meaning and minor fix

## Additional information

List all affected URLs 

https://devdocs.magento.com/guides/v2.3/inventory/
https://devdocs.magento.com/guides/v2.3/release-notes/ReleaseNotes2.3.0OpenSource.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Magento Commerce 2.3.0 includes a wealth of new features as well as hundreds of
 
 * **Bulk Web APIs**  allow all existing REST APIs to accept payloads with multiple entities. These community-contributed bulk APIs support more efficient and scalable implementations that eliminate round-trip network overhead. Like asynchronous APIs, bulk web APIs can be used in conjunction with queues that have also been migrated to Magento Open Source. [See Bulk endpoints](https://devdocs.magento.com/guides/v2.3/rest/bulk-endpoints.html) for more information. 
 
-* **Updates to Magento's tech stack (including upgraded PHP support to maintain PCI compliance)** include upgrades to Redis, MySQL, Elasticsearch, compatibility with PHP 7.2.x. 
+* **Updates to Magento's tech stack (including upgraded PHP support to maintain PCI compliance)** include upgrades to Redis, MySQL, Elasticsearch and compatibility with PHP 7.2.x.
```
Comment (shrielenee) [inline guides/v2.3/release-notes/ReleaseNotes2.3.0Commerce.md]: We use the Oxford, or serial, comma in a series. So in this instance, there would be a comma after "Elasticsearch" too. I will make this change but please note it for the future.

**Comment by shrielenee** [comment]
running tests

**Comment by shrielenee** [comment]
running tests

**Pull request merged**